docs/agents/sematext-agent/kubernetes/metrics.md

```diff
@@ -1,42 +1,763 @@
+|kubernetes.pod.count.running| gauge | long |equal to one if the pod has been scheduled on a node and at least one of his containers is running|
+
+### Deployment Metrics
+
+|Name|Type| Unit   | Label | Description          |
+|----|----|--------|-------|----------------------|
+|kubernetes.deployment.count|gauge|  | deployment count | number of active deployments |
+|kubernetes.deployment.replicas|gauge|  | replica count | number of active replicas |
+|kubernetes.deployment.replicas.avail|gauge|  | number of available replicas. Replicas are marked as available if they are passing the health check |
+|kubernetes.deployment.replicas.desired|gauge|  | number of desired replicas as defined in the deployment |
+
+### DaemonSet Metrics
+|Name|Type| Unit   | Description                  |
+|----|----|--------|-------|----------------------|
+|kubernetes.daemonset.number.available|gauge| long | number of nodes that should be running the daemon pod |
+|kubernetes.daemonset.number.misscheduled|gauge| long | number of nodes that are not supposed to run the daemon pod |
+|kubernetes.daemonset.number.ready|gauge| long | number of nodes that have one or more of the daemon pods running and ready |
+|kubernetes.daemonset.number.unavailable|gauge| long | number of nodes that have none of the daemon pods running and available |
+|kubernetes.daemonset.scheduled.updated|gauge| long |  number of nodes that are running updated daemon pod |
+|kubernetes.daemonset.scheduled.current|gauge| long | number of nodes that are running at least 1 daemon pod |
+|kubernetes.daemonset.scheduled.desired|gauge| long | number of nodes that should be running the daemon pod  |
+
+### StatefulSet Metrics
+|Name|Type| Unit   | Description                  |
+|----|----|--------|-------|----------------------|
+|kubernetes.statefulset.desired.replicas|gauge| long | number of desired replicas |
+|kubernetes.statefulset.replicas|gauge| long | number of active replicas |
+|kubernetes.statefulset.replicas.current|gauge| long | number of current replicas |
+|kubernetes.statefulset.replicas.updated|gauge| long | number of updated replicas |
+|kubernetes.statefulset.replicas.ready|gauge| long | number of Pods that have a Ready Condition |
+|kubernetes.statefulset.revision.update|gauge| long | update revision |
+|kubernetes.statefulset.revision.current|gauge| long | the current version of the StatefulSet used to generate Pods |
+
+
+### CronJob Metrics
+
+| Metric Name                     | Type    | Unit      | Description             |
+|---------------------------------|---------|-----------|-------------------------|
+| kubernetes.cronjob.name         | tag     |           | Cronjob name            |
+| kubernetes.cronjob.cluster.name | tag     |           | cluster name            |
+| kubernetes.cronjob.namespace    | tag     |           | namespace               |
+| kubernetes.cronjob.scheduled    | tag     |           | schedule setup  [ * 1 * * ] |
```

Comment:
We don't put tags in this document. We should include all tags for all the other metrics (which would be useful) or remove the tags from here

Reply:
Correct!